quadrat/inc/customization.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+require_once 'wp-customize-global-styles-setting.php';
+
+class GlobalStylesCustomizer {
+
+	private $palette_colors;
+
+	function __construct() {
+		add_action( 'customize_register', array( $this, 'parse_theme_palette_colors' ) );
+
+		add_action( 'customize_register', array( $this, 'register_section' ) );
+
+		/* Customizer Preview JS */
+		add_action( 'customize_preview_init', array( $this, 'customize_preview_js' ) );
+		add_action( 'wp_enqueue_scripts', array( $this, 'create_customization_style_element' ) );
+	}
+
+	function create_customization_style_element() {
+		//NOTE: The original implementation wasn't quite this convelouted (there was no noop.css file, etc) but I couldn't get it to work otherwise.
```

Comment:
```suggestion
		//NOTE: The original implementation wasn't quite this convoluted (there was no noop.css file, etc) but I couldn't get it to work otherwise.
```
Just a typo